### ISSUE TITLE
Queue removeFinishedJobs option may cause errors

### ISSUE THREAD

**Issue by steindani** [grantcarthew/node-rethinkdb-job-queue#26]
Although the [documentation says](https://github.com/grantcarthew/node-rethinkdb-job-queue/wiki/Queue-Options#queue-removefinishedjobs-option) that the `removeFinishedJobs` option can be set to true, using it returns error messages.

```
r.db("DB").table("TABLE").orderBy({
    index: "indexFinishedDateFinished"
}).filter(r.row("dateFinished").add(r.expr(true).div(1000)).lt(r.now())).delete()
                                    ^^^^^^^^^^^^^^^^^^^^^^
```

RethinkDB version: 2.3.5

**Comment by grantcarthew** [comment]
Hi @steindani, confirmed bug. Thanks for the heads up.
I'm working on it when I can.

**Comment by grantcarthew** [comment]
This is fixed in master now @steindani, I am just working on tests. Once this and #28 are complete I will publish v1.0.1.

**Comment by grantcarthew** [comment]
Fixed in v1.0.1.
Thanks @steindani

**Comment by steindani** [comment]
Awesome, thanks, @grantcarthew!
